### PR TITLE
Downcase email in pending invitations to join a group

### PR DESCRIPTION
**Pull request by jadevdl**

### CODE REVIEW

File: test/integration/pending_invitations_test.rb [hunk 1]
```diff
@@ -170,6 +198,7 @@ class PendingInvitationsTest < ActionDispatch::IntegrationTest
     as_user(account)
 
     visit invitation.link(host_and_port)
+    binding.pry
```
Comment (hchevalier) [inline test/integration/pending_invitations_test.rb:201]: Guess you forgot to remove this one :)

**Comment by hchevalier** [comment]
I think it will be necessary to also downcase email in the SessionController in case people try to log with My.Email@MyDomain.com.

Thanks a lot to tackle this issue by the way!

File: test/integration/pending_invitations_test.rb (hunk 1 above)
Comment (jadevdl) [reply test/integration/pending_invitations_test.rb]: oops 😄

**Comment by jadevdl** [comment]
Sure! I added two cases:

- user tries to create an account with an uppercased email
- user tries to connect using an uppercased email

do you think of other cases?

**Comment by hchevalier** [comment]
There is a last case for users logging in through their Google account in Account model (`from_omniauth` class method) ;)

After that, I guess every cases will be covered!

**Comment by jadevdl** [comment]
I added the `downcase` but I don't know how to test it (and if it's necessary) :)

**Review by hchevalier — APPROVED** [review]
Great!

**Pull request merged**